app/views/compute_resources_vms/form/ovirt/_network.html.erb

```diff
@@ -3,3 +3,5 @@
 <%= select_f f, :network, compute_resource.networks(selected_cluster ? { :cluster_id => selected_cluster } : { }), :id, :name,
              { }, :disabled => !new_vm, :class => "ovirt_network",
              :label         => _('Network'), :label_size => "col-md-3" %>
+<%= select_f f, :interface, [OpenStruct.new({:id =>'virtio', :name =>'VirtIO'}), OpenStruct.new({:id =>'rtl8139', :name =>'rtl8139'}), OpenStruct.new({:id =>'e1000', :name =>'e1000'}), OpenStruct.new({:id =>'pci_passthrough', :name =>'PCI Passthrough'})],
```

Comment:
Where does this list come from? is it possible that there will be other/different interface types in the future? in vmware we have a `nictype` list in the model, perhaps we should be consistent and use the same? 
Also, I don't see any handling on the server side, where is this handled?

Reply:
@tbrisker the list comes from ovirt, it is possible that in the future ovirt decided they want to support another interface type, but I don't see how can we get this list from ovirt through fog-ovirt.
about the server side, it's already implemented.

Reply:
I think this is fine, however @shiramax can you move it to a helper? I also think there are a few items missing, from what I see on https://www.rubydoc.info/gems/ovirt-engine-sdk/OvirtSDK4/NicInterface or https://docs.ansible.com/ansible/devel/modules/ovirt_nics_module.html

@tbrisker This is handled at `app/models/compute_resources/foreman/model/ovirt.rb#new_vm`, the `interface_attributes` are passed directly.